api/helpers/gamestate/moves/resolve/validate.js

```diff
@@ -40,13 +40,9 @@ module.exports = {
         throw new Error('You cannot resolve unless there is a one-off pending');
       }
 
-      // Must be your chance to counter
-      // It's your chance if it is your turn and there are an odd number of twos
-      // or if it's your opponent's turn and there are an even number of twos
-      // (since the opponent of the person who played the one-off counters first)
-      const isYourTurn = currentState.turn % 2 === playedBy;
-      const numTwosIsEven = currentState.twos.length % 2 === 0;
-      if (isYourTurn === numTwosIsEven) {
+      // Must be your chance to resolve
+      const yourTurnToResolve = sails.helpers.gamestate.yourTurnToCounter(currentState, playedBy);
+      if (!yourTurnToResolve) {
```

Comment:
Pulled this into a helper since it's used in resolve/validate and counter/validate